### ISSUE TITLE
Lesson 15: import url

### ISSUE THREAD

**Issue by nlviands** [KalobTaulien/django-201#1]
I know you are probably busy with other things and this is a couple of years old but could you update the code to reflect that django.conf.urls.url() was deprecated in Django 3.0, and is removed in Django 4.0+.  

as of lesson 15, is this what urlpatterns would look like:

urlpatterns = [
    path("admin/", admin.site.urls),
    path("", include(feed_urls, namespace="feed")),
    re_path("",include("allauth.urls")),

]

Thanks

**Comment by alphad01** [comment]
Yes, django.conf.urls.url() was deprecated in Django 3.0, and is removed in Django 4.0+. Importing re_path as url should solve the problem.

from django.urls import re_path as url

urlpatterns = [
    path('admin/', admin.site.urls),
    path("", include(feed_urls, namespace="feed")),
    url("", include("allauth.urls")),
]
